src/core/include/openvino/core/descriptor/tensor.hpp

```diff
@@ -32,22 +32,12 @@ namespace descriptor {
 
 class Tensor;
 
-OPENVINO_DEPRECATED("get_ov_tensor_legacy_name() is deprecated. Please don't use this function.")
-OPENVINO_API
-std::string get_ov_tensor_legacy_name(const Tensor& tensor);
-
-OPENVINO_DEPRECATED("set_ov_tensor_legacy_name() is deprecated. Please don't use this function.")
-OPENVINO_API
-void set_ov_tensor_legacy_name(Tensor& tensor, const std::string& tensor_name);
-
```

Comment:
@ilya-lavrenov  @vurusovs  For set_ov_tensor_legacy_name()/get_ov_tensor_legacy_name() deprecated, which means we do not need this API & the class member m_legacy_name directly, right?

Reply:
yes